tests/test_eload_preparation.py

```diff
@@ -99,7 +100,20 @@ def test_find_genome_single_sequence(self):
         cfg.content['eutils_api_key'] = None
         self.eload.eload_cfg.set('submission', 'scientific_name', value='Thingy thingus')
         self.eload.eload_cfg.set('submission', 'analyses', 'Analysis alias test', 'assembly_accession', value='AJ312413.2')
-        self.eload.find_genome()
-        assert self.eload.eload_cfg['submission']['analyses']['Analysis alias test']['assembly_fasta'] \
-               == 'tests/resources/genomes/thingy_thingus/AJ312413.2/AJ312413.2.fa'
-        assert 'assembly_report' not in self.eload.eload_cfg['submission']
+
+        with mock.patch("eva_submission.eload_preparation.requests.put", return_value=Mock(status_code=200)):
```

Comment:
For consistency and cleaner imports, I'd use either `patch` and `Mock`, or `mock.patch` and `mock.Mock`, but not a mixture of both.  And then remove the unnecessary import statements.

Reply:
This is implemented as well in the new changes